### PR TITLE
Add a sample to play with Object Recognition Kitchen(ORK)

### PR DESCRIPTION
**Pull request by yoneken**
This PR adds a sample to use [ORK](http://wg-perception.github.io/object_recognition_core/index.html#index) with fetch.
This sample is confirmed with gazebo and actual robot.

In gazebo,
`roslaunch fetch_gazebo pickplace_playground.launch`
[Another terminal]
`roslaunch fetch_gazebo_demo recognize_demo.launch`

Enjoy!

### CODE REVIEW

**Comment by moriarty** [comment]
Thanks, and sorry this PR has been neglected for so long.

**Indigo** is about to hit **EOL** do you know if this works in **Melodic**?

File: fetch_gazebo_demo/config/detection.fetch.ork [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    th_obj_dist: 0.1
+    icp_dist_min: 0.05 #0.06
+    px_match_min: 0.25 #0.5
+    depth_frame_id: 'head_camera_depth_optical_frame' #CameraDepth_frame
+    # The list of object_ids to analyze
+    object_ids: 'all'
+    #object_ids: ['24e0dc763c918714368b41b1c3000c23']
+    db:
+      type: 'CouchDB'
+      root: 'http://localhost:5984'
```
Comment (moriarty) [inline fetch_gazebo_demo/config/detection.fetch.ork:66]: I assume "object_recognition_core" has setup this **CouchDB** instance in it's install? or is that in the ORK install document

**Review by moriarty — COMMENTED** [review]
I haven't used the Object Recognition Kitchen.

This needs a little more documentation, and maybe some setup instructions.
It could also have a tutorial page added to https://docs.fetchrobotics.com/ which is hosted at https://github.com/fetchrobotics/docs

**Comment by yoneken** [comment]
I'm sorry but I left from the development for Fetch robots.
So, I cannot confirm your request any further.
Please feel free to close this PR.

**Comment by moriarty** [comment]
@yoneken thanks for the contribution and sorry it didn't get attention sooner. I've closed this for now.

**Pull request closed without merging**